pom.xml

```
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <artifactId>fi.metatavu.keycloak</artifactId>
+    <groupId>otp.authenticator</groupId>
+    <version>1.0-SNAPSHOT</version>
```

I believe you meant this to be fi.metatavu.keycloak:otp-authenticator:1.0.0-SNAPSHOT
```suggestion
    <groupId>fi.metatavu.keycloak</groupId>
    <artifactId>otp-authenticator</artifactId>
    <version>1.0.0-SNAPSHOT</version>
```

---

Resolved